From 59fce95047d598ca78587544d2af23bd8eab862e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Rainer=20M=C3=BCller?= <raimue@codingfarm.de>
Date: Wed, 20 Feb 2013 19:09:54 +0100
Subject: [PATCH 2/4] watch: Remove unused header

No declaration from proc/procps.h is actually used for watch.
---
 watch.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/watch.c b/watch.c
index 5a22952..0713448 100644
--- a/watch.c
+++ b/watch.c
@@ -31,7 +31,6 @@
 #include "config.h"
 #include "fileutils.h"
 #include "nls.h"
-#include "proc/procps.h"
 #include "strutils.h"
 #include "xalloc.h"
 #include <ctype.h>
-- 
1.8.1.3

